src/runner.ts

```diff
@@ -415,13 +435,13 @@ export class ScreenshotRunner {
         }
 
         let time = 0;
-        while (error === null && eventCount < count && time < timeout) {
+        while (eventCount < count && time < timeout) {
             const debounceTime = 1000;
             await new Promise((res) => setTimeout(res, debounceTime));
             time += debounceTime;
         }
 
-        if (error !== null) {
+        for (const error of errors) {
```

Comment:
I guess this is not required since we print them in onerror? I don't know why I did that in the first place